test/server.js

```diff
@@ -1,32 +1,32 @@
-const expect = require('expect');
-const request = require('supertest');
-const createServer = require('../lib/server');
+const expect = require('expect')
+const request = require('supertest')
+const createServer = require('../lib/server')
 
 describe('server', function () {
-  let server;
-  let webhook;
+  let server
+  let webhook
 
   beforeEach(() => {
-    webhook = expect.createSpy().andCall((req, res, next) => next());
-    server = createServer(webhook);
-  });
+    webhook = expect.createSpy().andCall((req, res, next) => next())
+    server = createServer(webhook)
+  })
 
   describe('GET /ping', () => {
     it('returns a 200 repsonse', () => {
-      return request(server).get('/ping').expect(200, 'PONG');
-    });
-  });
+      return request(server).get('/ping').expect(200, 'PONG')
+    })
+  })
 
   describe('webhook handler', () => {
     it('should 500 on a webhook error', () => {
-      webhook.andCall((req, res, callback) => callback('webhook error'));
-      return request(server).post('/').expect(500);
-    });
-  });
+      webhook.andCall((req, res, callback) => callback('webhook error')) // eslint-disable-line standard/no-callback-literal
```

Comment:
standard doesn't like callbacks with a string/literal. I was also conflicted about adding the disable vs doing
```js
webhook.andCall((req, res, callback) => callback(new Error('webhook error')))
```
I think creating real errors is possibly better practice?